apps/web/src/components/Explore/ExploreUserCard.tsx

```diff
@@ -98,6 +87,15 @@ export default function ExploreUserCard({ userRef, queryRef }: Props) {
     return badges;
   }, [user.badges]);
 
+  const isMobile = useIsMobileWindowWidth();
+
+  const bioFirstLine = useMemo(() => {
+    if (!user.bio) {
```

Comment:
could we make sure to use the [unescape util](https://github.com/gallery-so/gallery/blob/main/apps/web/src/scenes/UserGalleryPage/UserNameAndDescriptionHeader.tsx#L79) here?

side note, in the future would be great to auto-unescape all string fields in the relay cache automatically. @tbezman do you know a simple way to do that? we use it in ~25 locations in the codebase and i think it'd be relatively harmless to do across the board

Reply:
ooh good catch. added unescape !

Reply:
Relay provides no mechanism to do things like this globally. Thinking of a few options here.

1. Have the backend escape these markdown fields for us.
2. Instead of using a `String` type, we use a custom scalar type to make it a little bit more obvious to the engineer that it has to be escaped although I'm not entirely sure what this would look like in practice.
3. Add a custom Relay Resolver. This is like adding a field to our schema that's only available on the client. Something like `escapedBio` that would just run the function for us. I don't think people would remember to use this though.